masonry/src/contexts.rs

```diff
@@ -704,3 +706,146 @@ impl AccessCtx<'_> {
         self.widget_state.needs_accessibility_update
     }
 }
+
+macro_rules! impl_get_raw {
+    ($SomeCtx:tt) => {
+        impl<'s> $SomeCtx<'s> {
+            /// Get a child context and a raw shared reference to a child widget.
+            ///
+            /// The child context can be used to call context methods on behalf of the
+            /// child widget.
```

Comment:
I notice that in the current code, the context in `RawWrapper` is never used. If the child cannot have been mutated, how meaningful even are these context methods?

Reply:
They have a few getters. But yes, I mostly include the context types for symmetry between the RawWrapper and RawWrapperMut.